docs/examples/model_repository/resnet_torchscript/Readme.md

```diff
@@ -0,0 +1,154 @@
+# Loading Pytorch Models To Nvidia Triton Inference Server
+
+Pytorch models are supported by the Nvidia [Triton Inference Server](https://github.com/triton-inference-server/). To Load the Pytorch model to triton inference server one must convert the model either ONNX format or in the TorchScript format.
+Here we will see how to convert the Pytorch Model to the Torchscript format and load it to Triton.
```

Comment:
convert a PyTorch model defined to TorchScript format and load/run it using Triton Inference Server.